recipes/python-rocksdb/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "python-rocksdb" %}
+{% set version = "0.7.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: aa692fbd06d8ed4ad641feee2970062fe01120e87b645a29403c71c628639e4f
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

`rocksdb` is not available for Windows, so you need to skip it (or fix the `rocksdb` recipe).
```suggestion
  skip: True  # [win]
  script: "{{ PYTHON }} -m pip install . -vv"
```